docs/lib/graphqlapi/fragments/ios/mutate-data.md

```diff
@@ -54,6 +67,14 @@ func updateTodo() -> AnyCancellable {
 }
 ```
 
+To create data, replace the request with `.create`
```

Comment:
Can we add similar to Android and Flutter? Or create a backlog issue to add these.

Reply:
yup, adding those in into this PR